src/2_RTB/VerbaleInterno_071123/content.typ

```diff
@@ -0,0 +1,85 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+= Sintesi Elaborazione Incontro
+
+== Assegnazione dei ruoli
+Su proposta dei componenti è stata assegnata questa prima distribuzione dei ruoli:
+- Responsabile: Riccardo Toniolo;
+- Amministratore: Matteo Rango;
+- Analista: Simone Caregnato;
+- Analista: Nancy Kalaj;
+- Verificatore: Riccardo Constantin;
+- Verificatore: Giacomo D'Ovidio;
+Per il momento i ruoli di Progettista e Programmatore non sono stati assegnati, in quanto non sono significativi per questa fase di progetto.
```

Comment:
riguarderà principalmente le prime 2-3 settimane credo. Mettere in questa fase sembra troppo lunga temporalmente